x-pack/plugins/ml/server/saved_objects/sync_task.ts

```diff
@@ -126,7 +126,7 @@ export class SavedObjectsSyncService {
 
       return taskInstance;
     } catch (e) {
-      this.log.error(`Error running task: ${SAVED_OBJECTS_SYNC_TASK_ID}, `, e?.message() ?? e);
+      this.log.error(`Error running task: ${SAVED_OBJECTS_SYNC_TASK_ID}, `, e?.message ?? e);
```

Comment:
Why was this change necessary?

Reply:
Nope, not really.
It's a bug in the code that prevents from logging the caught error though, and causes a new Exception to be thrown.
I uncovered it during some tests.

I can create a separate PR if needed. @elastic/ml-ui WDYT?

Reply:
This looks like a typo.
I'm happy for this to be fixed in this PR, no need to create a new one.
Thanks for spotting and fixing it.